src/postamble.js

```diff
@@ -125,6 +125,10 @@ Module['then'] = function(func) {
 };
 #endif
 
+/**
```

Comment:
Are you sure? When I run `emcc hello_world.c --js-opts 2 --llvm-opts 2 --minify 0` it does strip comments, as it has run the js optimizer which doesn't know how to preserve them.